projects/1.18/assets/architects-palette/architects_palette/lang/zh_cn.json

```
@@ -12,22 +12,22 @@
 
 
   "__comment__": "These strings are used in the JEI/REI plugins.",
-  "architects_palette.info.warping_recipe_title": "跃迁",
+  "architects_palette.info.warping_recipe_title": "移形",
   "__comment__": "EMI implementation detail. Should be the same string as above.",
-  "emi.category.architects_palette.warping": "跃迁",
+  "emi.category.architects_palette.warping": "移形",
   "__comment__": "The %s in the string is replaced with the corresponding dimension when displayed in game",
   "architects_palette.info.warping_toss_description": "将其扔进一个连通至%s的传送门",
   "__comment__": "Modders can add support to the preview by adding something like this to their lang files, under the architects_palette namespace",
   "architects_palette.info.dimension.minecraft.the_nether": "下界",
   "architects_palette.info.dimension.minecraft.the_end": "末地",
   "architects_palette.info.dimension.minecraft.overworld": "主世界",
   "architects_palette.info.chiseled_chargeable": "在其中放入海洋之心以给它和所有相邻同类方块供能。",
-  "architects_palette.info.chargeable": "这些方块能从錾制渊晶石砖中汲取能量，更多信息请查询錾制渊晶石砖。",
-  "architects_palette.info.totem_carving": "在图腾上使用斧以切换图腾样式。",
+  "architects_palette.info.chargeable": "这些方块能从对应的錾制方块中汲取能量，更多信息请查询对应錾制方块。",

```

原文`These blocks can receive power from their Chiseled Bricks, check them out for more info.`
测试下是錾制方块还是錾制砖块变种

---

是錾制砖块，不过对应的方块系列里只有`錾制xx砖`这个符合条件的
```suggestion
  "architects_palette.info.chargeable": "这些方块能从对应的錾制砖块中汲取能量，更多信息请查询对应錾制砖块。",
```